jarviscli/plugins/caesar_cipher.py

```diff
@@ -4,7 +4,7 @@
 
 
 @plugin("caesar cipher")
-def caesar_cipher_converter(jarvis, str):
+def caesar_cipher_converter(jarvis):
```

Comment:
```suggestion
def caesar_cipher_converter(jarvis, s):
```

Second parameter is required (but you are right, naming a parameter str is not good).